package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "prepare": "husky install",
     "test": "pnpm nx run extension:test",
-    "test-watch": "pnpm nx run extension:test-watch"
+    "test-watch": "pnpm nx run extension:test-watch",
+    "deploy:vscode": "vsce publish",
+    "deploy:ovsx": "ovsx publish"
```

Comment:
I'm wondering if it would be cleaner to have this only under `app/vscode` and potentially as a nx task (vs in the package file)?

Reply:
Advantage of nx : building dependencies automatically

Reply:
Good point, I'll move them here